client/src/app/containers/event-card/EventCard.js

```diff
@@ -89,20 +76,10 @@ function EventCard(props) {
               <Moment format='h:mma'>{props.end_date}</Moment>
             </div>
           </p>
-          <button className='popup-button-event-card'>
-            <a
-              href={props.link}
-              target='_blank'
-              rel='noopener noreferrer'
-              className='popup-button-event-card-link'
-            >
-              <span>View on Facebook</span>
-            </a>
-          </button>
         </div>
         <div className='desc-text-wrapper-event-card'>
           <p className='desc-text-event-card'>
-            <Linkify> {props.description}</Linkify>
+            <div dangerouslySetInnerHTML={{ __html: props.description }} />
           </p>
         </div>
         <div className='modal-bottom-part'>
```

Comment:
Nitpick: would prefer to have a transition on modal open and close as well as button hover, but if it's too much, no worries.

Reply:
Issue: The modal font isn't Poppins. Was there a reason for keeping it as the Bootstrap default?

Reply:
Issue: the modal scrollbar is somewhat hard to see due to low contrast. Is it possible to configure it?
<img width="385" alt="image" src="https://github.com/HackAtUCI/Club-Site/assets/28026520/2dc63643-c9ed-495b-9145-34864eeea17a">

Reply:
While I agree these are important lingering design issues, they are outside the scope of the data ingestion changes happening with this PR. Let's create a separate issue for improving the EventCard modal design.